<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGameWindow</class>
 <widget class="QWidget" name="CGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CGameWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {border: 1px dashed black;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="1">
    <widget class="CBoard" name="board">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="playerAreaWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="playerAreaHeader">
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;b&gt;Your cards:&lt;/b&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="playerAreaMinimizeButton">
        <property name="toolTip">
         <string>Minimize card toolbar</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>â—„</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QWidget" name="playerCards" native="true">
        <layout class="QVBoxLayout" name="playerCardsVBLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="gameStatusWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1024</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="blue_ico">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p align=&quot;justify&quot;&gt;Number of blue disease cubes left in the reserves.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../pandemic.qrc">:/icons/img/icons/cube_blue.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="blue_info">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p align=&quot;justify&quot;&gt;Number of blue disease cubes left in the reserves.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>x28</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="yellow_ico">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p align=&quot;justify&quot;&gt;Number of yellow disease cubes left in the reserves.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../pandemic.qrc">:/icons/img/icons/cube_yellow.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="yellow_info">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p align=&quot;justify&quot;&gt;Number of yellow disease cubes left in the reserves.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>&lt;b&gt;x28&lt;/b&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="black_ico">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p align=&quot;justify&quot;&gt;Number of black disease cubes left in the reserves.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../pandemic.qrc">:/icons/img/icons/cube_black.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="black_info">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p align=&quot;justify&quot;&gt;Number of black disease cubes left in the reserves.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>&lt;b&gt;x28&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="red_ico">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p align=&quot;justify&quot;&gt;Number of red disease cubes left in the reserves.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../pandemic.qrc">:/icons/img/icons/cube_red.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="red_info">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p align=&quot;justify&quot;&gt;Number of red disease cubes left in the reserves.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>&lt;b&gt;x28&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="base_ico">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p align=&quot;justify&quot;&gt;Number of research stations left in the reserves.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../pandemic.qrc">:/icons/img/icons/base.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="base_info">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p align=&quot;justify&quot;&gt;Number of research stations left in the reserves.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>&lt;b&gt;x4&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cards_ico">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p align=&quot;justify&quot;&gt;Number of player cards left in the deck.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../pandemic.qrc">:/icons/img/icons/card.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cards_info">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p align=&quot;justify&quot;&gt;Number of player cards left in the deck.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>&lt;b&gt;x40&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="outbreak_ico">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p align=&quot;justify&quot;&gt;Number of outbreaks that have already occured.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../pandemic.qrc">:/icons/img/icons/outbreaks.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="outbreak_info">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p align=&quot;justify&quot;&gt;Number of outbreaks that have already occured.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>&lt;b&gt;x3&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="infections_ico">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p align=&quot;justify&quot;&gt;&lt;b&gt;Infection Rate&lt;/b&gt; - number of Infection Cards to be drawn in &lt;i&gt;Infector Phase&lt;/i&gt;.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../pandemic.qrc">:/icons/img/icons/outbreaks.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="infections_info">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;p align=&quot;justify&quot;&gt;&lt;b&gt;Infection Rate&lt;/b&gt; - number of Infection Cards to be drawn in &lt;i&gt;Infector Phase&lt;/i&gt;.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>&lt;b&gt;x2&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cureStatus_blue">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../pandemic.qrc">:/icons/img/icons/cure_blue_new.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cureStatus_yellow">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../pandemic.qrc">:/icons/img/icons/cure_yellow_new.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cureStatus_black">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../pandemic.qrc">:/icons/img/icons/cure_black_new.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cureStatus_red">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../pandemic.qrc">:/icons/img/icons/cure_red_new.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CBoard</class>
   <extends>QLabel</extends>
   <header>CBoard.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pandemic.qrc"/>
  <include location="../cards.qrc"/>
 </resources>
 <connections/>
</ui>
